primary/Tenkuu Danzai Skelter Heaven (primary).ass

```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
 Dialogue: 0,0:14:13.50,0:14:15.93,Default,,0,0,0,,Rin! What are you doing?
-Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I didn't follow your orders all this time because of how I was created!
-Dialogue: 0,0:14:20.72,0:14:23.94,Default,,0,0,0,,It was because I loved you!
+Dialogue: 0,0:14:15.93,0:14:20.34,Default,,0,0,0,,I haven't followed your orders just \Nbecause I was created to do so!
+Dialogue: 0,0:14:20.72,0:14:23.94,Default,,0,0,0,,I've done it because I love you!
 Dialogue: 0,0:14:26.71,0:14:27.95,Default,,0,0,0,,That's right.
 Dialogue: 0,0:14:28.91,0:14:30.92,Default,,0,0,0,,I'm going in first!
 Dialogue: 0,0:14:36.15,0:14:37.18,Default,,0,0,0,,Don't!
 Dialogue: 0,0:14:37.86,0:14:39.65,Default,,0,0,0,,Stop it!
 Dialogue: 0,0:14:58.01,0:14:59.23,Default,,0,0,0,,I got through!
 Dialogue: 0,0:15:01.13,0:15:03.18,Default,,0,0,0,,Here we go, Rin!
-Dialogue: 0,0:15:06.22,0:15:09.96,Default,,0,0,0,,Skelter Titan and Skelter Heaven have penetrated the target.
-Dialogue: 0,0:15:11.72,0:15:12.86,Default,,0,0,0,,What on earth...
-Dialogue: 0,0:15:13.58,0:15:14.34,Default,,0,0,0,,What is this?
-Dialogue: 0,0:15:21.98,0:15:24.14,Default,,0,0,0,,This is the controlling core.
-Dialogue: 0,0:15:24.95,0:15:28.03,Default,,0,0,0,,It's speaking directly into my mind
-Dialogue: 0,0:15:31.44,0:15:35.38,Default,,0,0,0,,Certainly, some humans are slaves to their emotions.
-Dialogue: 0,0:15:35.38,0:15:39.04,Default,,0,0,0,,And their history is one of destruction, no end in sight.
+Dialogue: 0,0:15:06.22,0:15:09.96,Default,,0,0,0,,Skelter Titan and Skelter Heaven \Nhave penetrated the target.
+Dialogue: 0,0:15:11.72,0:15:14.34,Default,joined,0,0,0,,{\i1}What on earth... {Line 2: What is this?}
+Dialogue: 0,0:15:21.98,0:15:24.14,Default,,0,0,0,,This is its logic{master} core.
+Dialogue: 0,0:15:24.95,0:15:28.03,Default,,0,0,0,,It's speaking directly into my mind.
+Dialogue: 0,0:15:31.44,0:15:35.38,Default,,0,0,0,,It's true some humans are slaves \Nto their emotions.
+Dialogue: 0,0:15:35.38,0:15:39.04,Default,,0,0,0,,Yes, their history is an endless chain of destruction.
 Dialogue: 0,0:15:40.22,0:15:44.46,Default,,0,0,0,,But that's only one side of humanity.
 Dialogue: 0,0:15:45.55,0:15:47.51,Default,,0,0,0,,We have to defeat you.
-Dialogue: 0,0:15:50.78,0:15:52.96,Default,,0,0,0,,But it's not for humanity.
-Dialogue: 0,0:15:52.96,0:15:55.62,Default,,0,0,0,,It's for the one person I want to protect!
-Dialogue: 0,0:15:55.62,0:15:57.44,Default,,0,0,0,,That's why I'm fighting!
+Dialogue: 0,0:15:50.78,0:15:52.96,Default,,0,0,0,,Not for the sake of mankind as a whole,
+Dialogue: 0,0:15:52.96,0:15:55.62,Default,,0,0,0,,but for the one person I care about most!
+Dialogue: 0,0:15:55.62,0:15:57.44,Default,,0,0,0,,I fight for that one man!
 Dialogue: 0,0:16:31.44,0:16:33.53,Default,,0,0,0,,Yes, that's exactly right.
```

Comment:
This feels more natural to me:
```suggestion
Dialogue: 0,0:16:31.44,0:16:33.53,Default,,0,0,0,,Yes, it is as you say.
```

Reply:
He's not responding to her, if that's what these changes are implying. From what I could tell, he's debriefing the situation to presumably someone higher up the food chain.

Reply:
The English of both changes can imply that he's responding to something that the person on the other side of the phone has said, and it must be a response to something because of the presence of "yes". That said, I can see how the first can be more of an assertion responding to clarification ("This thing happened." -> "Did it really?" -> "Yes, it really did."), whereas the second may imply more that he's receiving information.

"That's exactly right" still seems awkward to me, so perhaps this is a better wording that preserves the nuances of the first?
```suggestion
Dialogue: 0,0:16:31.44,0:16:33.53,Default,,0,0,0,,Yes, that is the case.
```